src/sources/data_source.js

```diff
@@ -45,7 +45,12 @@ export default class DataSource {
         // overzoom will apply for zooms higher than this
         this.max_zoom = (config.max_zoom != null) ? config.max_zoom : Geo.default_source_max_zoom;
 
+        // set a custom extra overzoom adjustment factor to load consistently lower zoom levels
+        // than the current map zoom level – eg a zoom_offset of 1 would load z3 data at z4
+        this.zoom_offset = (config.zoom_offset != null) ? config.zoom_offset : 0;
```

Comment:
We should enforce this is 0 or positive (no negative values), and log a warning with the `once` option ([example](https://github.com/tangrams/tangram/blob/548ebf28916fa0e77b67bb6204c555e6300bd8ca/src/styles/style.js#L272))